src/interfaces/IERC7540.sol

```diff
@@ -308,6 +308,12 @@ interface IERC7540Vault is
     /// @dev    MUST be called by endorsed sender
     function setEndorsedOperator(address owner, bool approved) external;
 
+    /// @notice TODO
+    function pricePerShare() external view returns (uint256);
+
+    /// @notice TODO
+    function priceLastUpdated() external view returns (uint64);
+
```

Comment:
Reminder: Use the `inheritdoc` under these method in `ERC7540Vault.sol`

Reply:
Not needed anymore actually now that the pool mgr is used